src/app/apps/gestor-usuarios/components/arbol-permisos/arbol-permisos-item.component.ts

```diff
@@ -138,7 +138,11 @@ export class ArbolPermisosItemComponent implements OnInit, OnChanges, AfterViewI
                         // [TODO] Buscar según el tipo
                         switch (this.item.type) {
                             case 'prestacion':
-                                this.conceptosTurneablesService.search({ ids: items }).subscribe((data) => {
+                                let srhPrest = { ids: items };
+                                if (this.item.subtype) {
+                                    srhPrest = Object.assign(srhPrest, { ambito: this.item.subtype } );
```

Comment:
Asignación más simple:
```suggestion
                                    srhPrest.ambito = this.item.subtype;
```